### ISSUE TITLE
Might be an issue with the radio button group

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab#223]
@zepumph, not sure what is causing VO over to miscount or double count the items parenting the radiobuttons. It might be a new VO bug

First step... Could you make sure the HTML is valid. I didn't check it.
![Screen Shot 2019-12-31 at 10 44 39 PM](https://user-images.githubusercontent.com/1902116/71637576-c42bc580-2c20-11ea-803b-50e6febfadeb.png)

**Comment by zepumph** [comment]
I cannot reproduce this on NVDA. @terracoda was this always the case, or did you have to use the keys a fair amount first? Anything reproduce-able about how you got into this state?

Here is the HTML, it looks valid to me, and our radio buttons haven't changed (that I'm aware of) in a long time:

```html
<ul data-trail-id="632-633-706-712-708-962-961-952-950" id="632-633-706-712-708-962-961-952-950" data-focusable="false"
    class="a11y-sibling">
  
  <li data-trail-id="632-633-706-712-708-962-961-952-950-928" id="container-632-633-706-712-708-962-961-952-950-928"
      data-focusable="false" class="a11y-sibling"><input data-trail-id="632-633-706-712-708-962-961-952-950-928"
                                                         id="632-633-706-712-708-962-961-952-950-928"
                                                         data-focusable="true" class="a11y-sibling" type="radio"
                                                         checked="checked" name="VerticalAquaRadioButtonGroup1"
                                                         style="top: 527.327px; left: 792.442px; width: 191.679px; height: 22.7083px;"><label
      data-trail-id="632-633-706-712-708-962-961-952-950-928" id="label-632-633-706-712-708-962-961-952-950-928"
      data-focusable="false" class="a11y-sibling" for="632-633-706-712-708-962-961-952-950-928"
      style="top: 527.327px; left: 792.442px; width: 191.679px; height: 22.7083px;">‪Decimal Notation‬</label></li>

  <li data-trail-id="632-633-706-712-708-962-961-952-950-938" id="container-632-633-706-712-708-962-961-952-950-938"
      data-focusable="false" class="a11y-sibling"><input data-trail-id="632-633-706-712-708-962-961-952-950-938"
                                                         id="632-633-706-712-708-962-961-952-950-938"
                                                         data-focusable="true" class="a11y-sibling" type="radio"
                                                         name="VerticalAquaRadioButtonGroup1"
                                                         style="top: 554.293px; left: 792.442px; width: 191.679px; height: 22.7083px;"><label
      data-trail-id="632-633-706-712-708-962-961-952-950-938" id="label-632-633-706-712-708-962-961-952-950-938"
      data-focusable="false" class="a11y-sibling" for="632-633-706-712-708-962-961-952-950-938"
      style="top: 554.293px; left: 792.442px; width: 191.679px; height: 22.7083px;">‪Scientific Notation‬</label></li>

  <li data-trail-id="632-633-706-712-708-962-961-952-950-948" id="container-632-633-706-712-708-962-961-952-950-948"
      data-focusable="false" class="a11y-sibling"><input data-trail-id="632-633-706-712-708-962-961-952-950-948"
                                                         id="632-633-706-712-708-962-961-952-950-948"
                                                         data-focusable="true" class="a11y-sibling" type="radio"
                                                         name="VerticalAquaRadioButtonGroup1"
                                                         style="top: 581.259px; left: 792.442px; width: 191.679px; height: 22.7083px;"><label
      data-trail-id="632-633-706-712-708-962-961-952-950-948" id="label-632-633-706-712-708-962-961-952-950-948"
      data-focusable="false" class="a11y-sibling" for="632-633-706-712-708-962-961-952-950-948"
      style="top: 581.259px; left: 792.442px; width: 191.679px; height: 22.7083px;">‪Hidden‬</label></li>
</ul>
```

**Comment by jessegreenberg** [comment]
I just tested with macOS VO in Safari and the radio buttons sound as I would expect. I hear "Decimal Notation, radio button, 1 of 3", "Scientific Notation, selected, radio button, 2 of 3" and so on.

I also just tested with iOS VO and it sounds good there too.

**Comment by zepumph** [comment]
I also could not reproduce on my iPad this morning either, I only ever had the VO highlight track to unchecked radio buttons, but never the phet highlight.

**Comment by zepumph** [comment]
@terracoda have you been able to discover steps to reproduce this?

**Comment by terracoda** [comment]
@zepumph, sorry for not being clear, this happens with keyboard, but only in a very limited case when reading with the cursor keys, though I haven't isolated it.

If I read straight through and it counts fine. I move my focus through and it is fine.

Sometimes if do something different and it has focus, I get the double count. It is like I do something (not sure what) to confuse VO.

I think this is a weird edge case which I can't figure out. QA, might have some insights when they test. Let's leave it to the expert testers!

**Comment by terracoda** [comment]
Just as aside, at first I thought it wasn't happening at all, and was going to happily close the issue,  and then I made it happen....but VO beeped at me, so I think I doing something atypical.

**Comment by terracoda** [comment]
@lmulhall-phet or @KatieWoe, coulsd you see if you can reproduce this on any "supported" devices.

I am using MacOs and have a bluetooth keyboard.

@jessegreenberg and @zepumph, cannot reproduce.

This does not happen on VO with gesture. I am using MacBook Pro High Sierra 10.13.6. It might just be me or my set-up.

The issue is not detrimental to the UX. The double counting goes away easily when focus is moved out of the radiogroup.

So, to try and reproduce, I move my focus to the radiobuttons and then try to read with the virtual cursor. Sometime I an make it read out a double count, but I can't figure out how.

Note that VO beeps at me sometimes, which makes me think I am doing something atypical for VO.

Don't spend too much time on this. As I said it is not detrimental to the experience and we want to get this into RC :-)

**Comment by terracoda** [comment]
Pleas close if this double counting is not easy to reproduce. Thanks :-)

**Comment by KatieWoe** [comment]
Will take a look after https://github.com/phetsims/QA/issues/465

**Comment by terracoda** [comment]
@KatieWoe, I am comfortable, if you are, just closing this issue and seeing if it comes up in the QA process? Is that a reasonable path?

**Comment by KatieWoe** [comment]
That sounds fine. Maybe leave open and link to the next rc so it becomes part of the test.

**Comment by terracoda** [comment]
That's a great idea! Thanks :-)

**Comment by loganbraywork** [comment]
This issue is easily reproducible in Chrome on MacOS 10.15. This issue does not appear on Safari on MacOS 10.15. We don't support or test VO on Chrome only Safari.

**Comment by zepumph** [comment]
Does that mean it should be closed as a wontfix?

**Comment by terracoda** [comment]
I think so @zepumph.

**Comment by terracoda** [comment]
Chrome has often counted things in a funny way.

**Comment by zepumph** [comment]
Alright sounds good thanks!

**Comment by terracoda** [comment]
@KatieWoe and @zepumph, FYI, in https://github.com/phetsims/ohms-law/issues/153#issuecomment-583528576, I found that this double counting consistently happens when reading a list of radiobuttons in reverse order with the cursor keys. 

I still don't think this needs to be fixed.
